packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -63,6 +64,25 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
     return await this.provider.wasm.contractQuery(address, input, params)
   }
 
+  signAndSend = async (messages: MsgExecuteContract[]): Promise<TransactionResponse> => {
+    try {
+      const tx = await this.wallet.createAndSignTx({
+        msgs: messages,
+        ...(this.wallet.key instanceof LedgerKey && {
+          signMode: SignMode.SIGN_MODE_LEGACY_AMINO_JSON,
+        }),
+      })
+
+      const res = await this.provider.tx.broadcast(tx)
+
+      logger.debug(res)
+      return this.wrapResponse(res)
+    } catch (e) {
+      const details = e.response.data
+      throw new Error(details.message)
```

Comment:
What's the purpose of this catch-throw?

Also if there is a reason to leave it in, we should use safe navigation:
```suggestion
      throw new Error(e.response.?data.?message)
```

Reply:
The default `e.message` gives almost no information. Adding the safe navigation